uw-frame-components/static.html

```diff
@@ -9,8 +9,8 @@
   }
 
   .slowfade {
```

Comment:
Eh, I'll remove it increase. I don't believe it has anything to do with the lag in responsiveness though. That seems to be due to something going on in the angularJS bootstrapping process.